multiversx_sdk/network_providers/transaction_decoder.py

```diff
@@ -106,9 +114,15 @@ def get_esdt_transaction_metadata(self, metadata: TransactionMetadata) -> Option
         result.value = value
         result.transfers = []
 
+        receiver = Address.new_from_bech32(result.receiver)
```

Comment:
How about the same case for NFT transfers or multi transfers? In that case, I think `receiver.is_smart_contract()` would not be appropriate.

Reply:
did the same for nft and multi transfer.